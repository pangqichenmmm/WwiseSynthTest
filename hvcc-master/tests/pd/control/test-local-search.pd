#N canvas 1506 476 243 165 10;
#X obj 49 84 abs/load_another;
