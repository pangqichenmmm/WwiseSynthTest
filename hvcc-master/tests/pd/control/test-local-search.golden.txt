[@ 0.000] print: it works
